### ISSUE TITLE
Deleting the neumes of a syllable with both `precedes` and `follows` makes the syllable preceding it unresponsive

### ISSUE THREAD

**Issue by JoyfulGen** [DDMAL/Neon2#1179]
At the moment, mei_encoding is producing syllables that have both a `precedes` and a `follows` attribute. This is not supposed to happen and I've made an issue about it here: https://github.com/DDMAL/Rodan/issues/1112.

However, while it's still happening it's creating a small bug within Neon. If you delete all the neumes of a syllable with both a `precedes` and a `follows` attribute, the syllable it follows becomes unresponsive.

In the file below, if you delete the syllable "hello" (a pale blue punctum on the right edge of the page), you won't be able to select the orange virga+divisio combo using the syllable selector anymore.
[168r for toggled syl.mei.zip](https://github.com/DDMAL/Neon/files/13976417/168r.for.toggled.syl.mei.zip)

@yinanazhou I think you should decide whether this actually needs to be fixed or not. This issue wouldn't exist if the problem wasn't in mei_encoding in the first place, so it could be left alone and forgotten as soon as mei_encoding is fixed. It's very mildly annoying, but I can always delete the neumes even if I can't delete the syllable, so it doesn't impede correction. Let me know what you think!

**Comment by yinanazhou** [comment]
Hi @JoyfulGen I've pushed an option in MEI Action called `untoggle invalid linked syllable`. Please let me know how you like it :)

**Comment by JoyfulGen** [comment]
@yinanazhou that's brilliant! I like this solution and it works very well. Thank you! You can close this issue now as far as I'm concerned.

**Comment by JoyfulGen** [comment]
Re-opening this issue because I've noticed two strange behaviours:
- It looks like if an uploaded file has a syllable with both a `precedes` and a `follows`, clicking on it doesn't highlight the syllables it's toggled to. Clicking on those syllables doesn't highlight the others either. 
- Using the "untoggle invalid toggled syllables" option does remove the `precedes` and `follows` tags, but the syllable doesn't get a `syl` element.

This is the file I've noticed this in: 
[164v.mei.zip](https://github.com/DDMAL/Neon/files/14537212/164v.mei.zip)

The id of the offending syllable is: m-22cf1a4f-cf1c-4959-8213-c75a27cf8a01
